src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVCustomReadFilter.java

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+import org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+/**
+ * Customized serializable reads filter, based on cmd line arguments provided, for structural variation purposes
+ */
+public final class SVCustomReadFilter implements ReadFilter {
+    private static final long serialVersionUID = 1L;
+
+    private static final PrimaryFilter pfilter = new PrimaryFilter();
+    private final CustomFilter sfilter;
+
+    public SVCustomReadFilter(final boolean[] sixFilterOptions,
```

Comment:
either pass these 6 arguments separately or make a new class for them. Passing a 6-element array is not good.